src/main/java/store/OrderItem.java

```diff
@@ -0,0 +1,62 @@
+package store;
+
+public class OrderItem {
+    private String name;
+    private int quantity;
+    private int price;
+    private int totalPrice;
+    private int payWithoutPromotionPrice;
+    private int promotionQuantity;
+
+    public OrderItem(String name, int quantity, int price, int totalPrice, int promotionQuantity) {
+        this.name = name;
+        this.quantity = quantity;
+        this.price = price;
+        this.totalPrice = totalPrice;
+        this.promotionQuantity = promotionQuantity;
+    }
+
+    public OrderItem(String name, int quantity, int price, int totalPrice, int payFromGeneralInventoryPrice, int promotionQuantity) {
```

Comment:
라인을 분리하면 가독성이 좋아질것 같아요!
```suggestion
    public OrderItem(
        String name,
        int quantity,
        int price, 
        int totalPrice, 
        int payFromGeneralInventoryPrice, 
        int promotionQuantity
    ) {
```

Reply:
> 라인을 분리하면 가독성이 좋아질것 같아요!

여지껏 생성자 코드를 작성할 때 한 줄에 나열하는 방식을 사용했는데, 말씀해주신대로 속성 단위로 구분해서
작성하니 보기 편한 것 같네요 :)

Reply:
> > 라인을 분리하면 가독성이 좋아질것 같아요!
> 
> 여지껏 생성자 코드를 작성할 때 한 줄에 나열하는 방식을 사용했는데, 보여주신 예시처럼 속성 단위로 구분해서 
작성하면 보기 편할 것 같네요 :)